### PR TITLE
[legacy] Set TextField's caret color to match textColor.

### PR DESCRIPTION
**Pull request by LaughingLeader**
Input text with a dark background and light text color should be more readable now.

The "text cursor" that appears when hovering over selectable text needs to reflect this next (the cursor currently becomes black).

This is in the Windows build.

### CODE REVIEW

**Comment by ubald** [comment]
Maybe this could be merged as a default value into https://github.com/openfl/lime/pull/320 it already adds support to customize caret and selection color.

**Comment by LaughingLeader** [comment]
Aha! Custom caret colors are even better. 

The "TextCursor" cursor color will definitely need to be addressed, as currently it defaults to black, making it hard to see with certain text/layouts colors.

**Comment by ubald** [comment]
yeah that one I ended up just doing my own openfl subclass, hiding the cursor and replacing it with one of my own as if I recall correctly it uses SDL cursors which are either white or black.

**Comment by LaughingLeader** [comment]
Nice. That's a good idea. I'll have to do that in the meantime.

**Comment by eminfedar** [comment]
Hey, anyone here? openfl.text.textField's caret isn't change his color when I set textColor to 0xFFFFFF.

Any solution?

File: legacy/project/src/common/TextField.cpp [hunk 1]
```diff
@@ -1512,7 +1517,11 @@ void TextField::Render( const RenderTarget &inTarget, const RenderState &inState
             double height = mLines[line].mMetrics.height;
             if (pos.y+height <= fieldHeight-GAP+1)
             {
-               mCaretGfx->lineStyle(1, 0x000000 ,1.0, false, ssmOpenGL  );
+				if(caretIndex == 0 && (mCharGroups.empty() || (mCharGroups.size() == 1 && mCharGroups[0]->Chars() == 0)) && getTextFormat()->align == tfaCenter)
```
Comment (Gama11) [inline legacy/project/src/common/TextField.cpp]: Looks like you're using tabs instead of spaces here.

**Pull request closed without merging**